Hello.py

```diff
@@ -1,8 +1,13 @@
+import argparse
+parser = argparse.ArgumentParser()
+parser.add_argument("host", help="the host for browsing")
```

Comment:
consider making this a non positional argument.  if more parameters are added it will allow things to be in any order.

Reply:
"wrong."